client/landing/stepper/declarative-flow/hosted-site-migration-v2/index.tsx

```diff
@@ -0,0 +1,23 @@
+import { translate } from 'i18n-calypso';
+import { type Flow } from 'calypso/landing/stepper/declarative-flow/internals/types';
+import { stepsWithRequiredLogin } from '../../utils/steps-with-required-login';
+import { STEPS } from '../internals/steps';
+
+export default {
+	name: 'hosted-site-migration-v2',
```

Comment:
> I didn't follow the /migration flow as we discussed just because we have another flow with the same name on the code. This flow name seems like a tech debit because it is actually the "import-focused" flow.
>
> We can have another issue just to rename it properly and fix the tech debit or user another name.

I tried to access the `/setup/migration` and it didn't load any flow. So I tried to use `"migration"` here and it worked. Is there other part that it conflicts that I'm missing?

Reply:
There are references here and more places that are using this const. 
https://github.com/Automattic/wp-calypso/blob/92cb036890e998c178a7ae036cd6b10419777e63/packages/onboarding/src/utils/flows.ts#L25

Reply:
Thank you for the context!
Issue created here: https://github.com/Automattic/wp-calypso/issues/93328.